keps/sig-cloud-provider/2395-removing-in-tree-cloud-providers/README.md

```diff
@@ -169,6 +192,149 @@ import (
 )
 ```
 
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+_This section must be completed when targeting alpha to a release._
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [X] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: DisableCloudProviders, DisableKubeletCloudCredentialProvider
```

Comment:
nit - can you please split them into two and list components for each of them separately?

Reply:
done, thanks